Makefile.cache

```diff
@@ -71,6 +71,8 @@ SONIC_COMMON_FLAGS_LIST :=  $(CONFIGURED_PLATFORM) \
                             $(CONFIGURED_ARCH) \
                             $(BLDENV) \
                             $(SONIC_DEBUGGING_ON) \
+							$(SONIC_VERSION_CONTROL_COMPONENTS) \
+							$(SONIC_VERSION_CACHE) \
```

Comment:
Please fix the TAB/SPACE alignment issue.

Reply:
Fixed